packages/test/test-end-to-end-tests/src/test/blobs.spec.ts

```diff
@@ -228,7 +228,7 @@ describeNoCompat("blobs", (getTestObjectProvider) => {
 
         await assert.rejects(
             container.attach(provider.driver.createCreateNewRequest(provider.documentId)),
-            /(0x206)|(0x202)/,
+            /(0x206)|(0x202)|(0x204)/,
```

Comment:
We should not rely on these short codes as they can change. Validating that container.attach throws should be enough.

Reply:
Aren't they stable once assigned? Only asserting that it throws leaves room for false negatives.

Reply:
AFAIK, they are not. @anthony-murphy or @vladsud can confirm if I am wrong.

Reply:
They are semi-stable. I believe coping code in same file will result in one of them (second in order?) to be replaced.